internal/cmd/mongodbflex/backup/update-schedule/update_schedule.go

```diff
@@ -21,10 +21,10 @@ import (
 const (
 	instanceIdFlag                     = "instance-id"
 	scheduleFlag                       = "schedule"
-	snapshotRetentionDaysFlag          = "save-snapshot-days"
-	dailySnapshotRetentionDaysFlag     = "save-daily-snapshot-days"
-	weeklySnapshotRetentionWeeksFlag   = "save-weekly-snapshot-weeks"
-	monthlySnapshotRetentionMonthsFlag = "save-monthly-snapshot-months"
+	snapshotRetentionDaysFlag          = "save-for-days"
```

Comment:
```suggestion
	snapshotRetentionDaysFlag          = "store-for-days"
```

Even if in the Portal is save, we should make it uniform in our CLI

Reply:
Yes, agreed. I missed that one